src/components/TodoList/TodoList.tsx

```diff
@@ -1,14 +1,47 @@
 /* eslint-disable */
 import React from 'react';
+import classNames from 'classnames';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { clearTodo, selectTodo } from '../../features/currentTodo';
+import { Todo } from '../../types';
 
 export const TodoList: React.FC = () => {
+  const todos = useAppSelector(state => state.todos)
+  const currentTodo = useAppSelector((store) => store.currentTodo);
+  const { query, status } = useAppSelector(state => state.filter);
+  
+  const dispatch = useAppDispatch();
+
+  const filteredTodo = todos.filter(todo => {
+    const matchesStatus = 
+    status === 'all' 
+      ? true 
+      : status === 'active'
+        ? !todo.completed
+        : todo.completed
+    const matchesQuery = todo.title.toLowerCase().includes(query.toLowerCase());
+
+    return matchesStatus && matchesQuery;
+  })
+
+  const handleCurrentTodo = (todo: Todo) => {
+    if (todo.id !== currentTodo?.id) {
+      dispatch(selectTodo(todo))
+    }
+    else {
+      dispatch(clearTodo())
+    }
+  }
+
+console.log()
```

Comment:
The `console.log()` statement is present but does not log any information. If it's not needed for debugging, consider removing it to clean up the code.